Assets/Global/Scripts/RoomLogic/Table.cs

```diff
@@ -140,6 +140,23 @@ private int GetIdFromOldRow(List<Row> existingBranches) {
         existingBranches.Remove(newRow);
         return newRow.id;
     }
+
+    public void PrintTableAsTree() {
+        Debug.Log("Generated Table (Tree Structure):");
```

Comment:
remove this or comment it for debugging purposes

Reply:
This whole method is a debug script, aka, 1 big Debug.Log() method. 
The only place where it is being used it is already commented out. But it is handy to still have this method in case you want to debug what is going wrong with the rooms.